opencodeblocks/graphics/blocks/blocksizegrip.py

```diff
@@ -6,22 +6,23 @@
 resize a block.
 """
 
+from PyQt5.QtCore import QPoint
 from PyQt5.QtWidgets import QGraphicsItem, QSizeGrip, QWidget
 from PyQt5.QtGui import QMouseEvent
 
 
 class BlockSizeGrip(QSizeGrip):
-    def __init__(self, container: QGraphicsItem, parent: QWidget = None):
+    def __init__(self, block: QGraphicsItem, parent: QWidget = None):
         """
             Constructor for BlockSizeGrip
 
-            container is the QGraphicsItem holding the QSizeGrip.
-            We should have: parent.graphicsProxyWidget().parent() == container
+            block is the QGraphicsItem holding the QSizeGrip.
+            It's usually an OCBBlock
```

Comment:
Why don't you type check with OCBBlock then ?

Reply:
This type of Grip could be used for other types of container. Currently, it's only used for OCBBlocks